File: android/src/main/java/com/getcapacitor/community/classes/events/PeerIDEvent.java
package com.getcapacitor.community.classes.events;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;
import java.util.UUID;

public class PeerIDEvent {

    @NonNull
    UUID peerID;

    public PeerIDEvent(@NonNull UUID peerID) {
        this.peerID = peerID;
    }

    @NonNull
    public JSObject toJSObject() {
        JSObject result = new JSObject();

        result.put("peerID", peerID.toString());

        return result;
    }
}

File: ios/Sources/BridgefyPlugin/Classes/Events/PeerIDEvent.swift
import Foundation
import Capacitor

@objc public class PeerIDEvent: NSObject {
    let peerID: UUID

    init(_ peerID: UUID) {
        self.peerID = peerID
    }

    public func toJSObject() -> JSObject {
        var result = JSObject()

        result["peerID"] = peerID.uuidString.lowercased()

        return result
    }
}
